### DISCUSSION TITLE
Note Size Search and Order by Sorting

### DISCUSSION THREAD

**Discussion by hl2guide** [standardnotes/forum#2743] [Suggestions]
Please consider adding the ability:

* to see the size of each note (KB or MB)
* to sort notes by size

Basic search syntax (size:>300KB) or size filtering (trackbar) would also be useful.

example: "Show us notes over 300KB in size"

Mainly used for keeping the app performant and loading in a reasonable time.

**Comment by atmoio** [comment]
Hmm, interesting idea. Could be useful. But we'd have to keep a dynamic index of note sizes so that the sorting and size computation is not on-demand.

One thing you could do now is to download a backup of your data. If encrypted, scan the JSON file for large items and then copy its UUID and paste in the search bar in SN and it will bring up that note. If decrypted, you should be able to sort notes by size in your file system.
